app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
       // load the max number of gene searches and corresponding error message.
       window.MAX_GENE_SEARCH = <%= Gene::MAX_GENE_SEARCH %>;
       window.MAX_GENE_SEARCH_MSG = '<%= Gene::MAX_GENE_SEARCH_MSG %>';
+      window.uniqueGenes = <%= @unique_genes.present? ? raw(@unique_genes) : [] %>;
 
```

Comment:
I'd like to start scoping all our scp-specific window variables that we use into a global 'SCP' object so they're easier to track and split out from other global.  So we could have a policy that all new window variables have to be inside window.SCP, and any time you mod code that uses an existing raw window variable, you move it inside window.SCP (assuming the migration isn't unduly difficult)

Reply:
since this is the first one, in application.js (or whatever you can guarantee will load first), you'll just add a `window.SCP = window.SCP || {}`